### PR TITLE
    Make key container creation idempotent

### PR DESCRIPTION
**Pull request by raildo**
Container creation returns a 409 Conflict if the container
    already exists, but since the request doesn't take any
    additional parameters, there's really no conflict.
    Update the API return, if the container exists, to 200 OK.
    Users could still distinguish whether the container
    already existed by checking for 200 OK vs 201 Created.
    
    Signed-off-by: Raildo Mascena <rmascena@redhat.com>
    Closes: #206

### CODE REVIEW

**Comment by simo5** [comment]
I do not get why lint failed ...

**Comment by raildo** [comment]
I fixed it in another PR, I push to this branch too, not it will works :)

File: src/custodia/client.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 from requests.adapters import HTTPAdapter
 from requests.compat import unquote, urlparse
 
+# pylint: disable=import-error
```
Comment (simo5) [inline src/custodia/client.py]: I am not sure this is the way to go

Comment (simo5) [inline src/custodia/client.py]: Can you put the lint fixes in a separate commit ?

Comment (raildo) [inline src/custodia/client.py]: sure, should I put the doc issue in a separate commit too?

Comment (simo5) [inline src/custodia/client.py]: yeah put all travis-related issues in one separate commit, please

Comment (raildo) [inline src/custodia/client.py]: I saw this case: https://github.com/latchset/custodia/blob/master/src/custodia/httpd/server.py#L20
and I thought that was something similar here.
